### PR TITLE
docs: release signature guides

### PR DESCRIPTION
**Pull request by luislhl**
### Acceptance Criteria
- Create guides on how to sign and verify releases
- Fix in the check_version script so it considers the package-lock.json file
- Remove section from README.md about a warning in Windows that was already fixed

### Reference:
- https://bitcoincore.org/en/download/
- https://gnupg.org/download/integrity_check.html

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -12,4 +12,79 @@ Create a git tag and a new release on GitHub.
...
+```
+gpg: Signature made Fri 09 Oct 2015 05:41:55 PM CEST using RSA key ID 4F25E3B6
+gpg: Good signature from "Werner Koch (dist sig)" [full]
+```
+
+Which indicates that the signature is valid.
+
+## Our public keys
+
+Current releases are signed by one or more of these keys:
```
Comment (msbrogli) [inline RELEASING.md:66]: Just add a link to a folder with all builders' PGP keys. So you can keep this file clean.

File: RELEASING.md [hunk 2]
```diff
@@ -12,4 +12,79 @@ Create a git tag and a new release on GitHub.
...
+```
+gpg: Can't check signature: No public key
+```
+
+If you did add our public keys, you may still get an error like this:
+
+TODO: Change the name in this example.
+
+```
+gpg: Good signature from "Werner Koch (dist sig)" [unknown]
```
Comment (msbrogli) [inline RELEASING.md:48]: I feel it's better to use an actual builder as an example in this doc.

File: RELEASING.md [hunk 3]
```diff
@@ -12,4 +12,79 @@ Create a git tag and a new release on GitHub.
 
 # Publishing the new App
```
Comment (msbrogli) [inline RELEASING.md:13]: Add instructions to generate and check the sha256sum of all files too?

**Review by msbrogli — CHANGES_REQUESTED** [review]

File: RELEASING.md (hunk 2 above)
Comment (luislhl) [reply RELEASING.md:48]: This was just a placeholder while I didn't get the actual builders.

File: RELEASING.md [hunk 4]
```diff
@@ -12,4 +12,79 @@ Create a git tag and a new release on GitHub.
...
-## TODO
\ No newline at end of file
+In case this is a Hathor release, make sure you also read our internal guide in https://github.com/HathorNetwork/ops-tools/blob/master/docs/release-guides/hathor-wallet.md
+
+1. Run the `release.sh` script, which will clean the environment and build the app for all platforms. The files go to the `dist` folder after the script finishes running. You should get 4 of them: `.AppImage`, `.deb`, `.dmg` and `.exe`.
+
+2. Sign all 4 files with the following command:
+
+```
+for file in *.AppImage *.deb *.dmg *.exe; do gpg --output "$file.sig" --detach-sig "$file"; done
```
[the remark this answers is not in the document]
Comment (luislhl) [reply RELEASING.md]: Yes, the new version includes this.

File: RELEASING.md (hunk 3 above)
Comment (luislhl) [reply RELEASING.md:13]: Done

File: RELEASING.md (hunk 1 above)
Comment (luislhl) [reply RELEASING.md]: Done

**Review by msbrogli — APPROVED** [review]

**Review by jansegre — APPROVED** [review]

**Pull request merged**